tests/languages/ini/section_feature.test

```diff
@@ -1,4 +1,4 @@
- [foo1] 
```

Comment:
Please revert this line. This tests that sections surrounded by spaces are still highlighted correctly. This is part of INI and used to be a bug IIRC.